kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/KuraIp6AddressGenerationMode.java

```diff
@@ -0,0 +1,44 @@
+package org.eclipse.kura.nm;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.kura.nm.enums.NMSettingIP6ConfigAddrGenMode;
+
+public enum KuraIp6AddressGenerationMode {
+
+    EUI64,
+    STABLE_PRIVACY;
+
+    private static final List<KuraIp6AddressGenerationMode> ADDRESS_GEN_MODE = Arrays
+            .asList(KuraIp6AddressGenerationMode.EUI64, KuraIp6AddressGenerationMode.STABLE_PRIVACY);
+
+    public static Boolean isEnabled(KuraIp6AddressGenerationMode status) {
+        return ADDRESS_GEN_MODE.contains(status);
+    }
```

Comment:
Why did you add all this `isEnabled` for the enums? Are they used somewhere? Aren't they somewhat redundant (especially in this case since **all** the enums mean that it `isEnabled`)?